include/tuvx/linear_algebra/linear_algebra.hpp

```diff
@@ -30,14 +30,18 @@ namespace tuvx
   /// @fn  Random vector function
   /// @brief Fill a std::vector with random values
   /// @param x Vector to allocate and fill
+  /// @param seed for random number generation
   template<typename T>
-  void FillRandom(std::vector<T> &x);
+  void FillRandom(std::vector<T> &x, const unsigned &seed);
 
   /// @fn Initialize random matrix function
   /// @brief Fills a matrix with uniformly distributed random values.
   /// @param A tridiagonal matrix to allocate and fill
+  /// @param seed for random number generation
+  /// @param make_diagonally_dominant make the tridiagonal matrix diagonally dominant (diagonal value is greater than sum of
+  /// the corrosponding row)
   template<typename T>
-  void FillRandom(TridiagonalMatrix<T> &A);
+  void FillRandom(TridiagonalMatrix<T> &A, const unsigned &seed, const bool &make_diagonally_dominant = false);
```

Comment:
We usually use `@breif` to explain functions/classes and structures, etc..  Instead of adding `@fn`, could you use  `@breif` to describe what the function does or is for? This request applies to other functions, classes, and structures. 
For consistency in [the comment style,](https://github.com/NCAR/tuv-x/blob/20e73de68b72c6ad8bbb7fd5be4ab102f4761397/include/tuvx/profile.hpp#L26-L36), I'd suggest

```suggestion
  /// @brief Fills a matrix with uniformly distributed random values.
  /// @param A Tridiagonal matrix to allocate and fill
  /// @param seed Seed for random number generation
  /// @param make_diagonally_dominant Flag that is set to true if matrix is diagonally dominant; false otherwise
  template<typename T>
  void FillRandom(TridiagonalMatrix<T> &A, const unsigned &seed, const bool &make_diagonally_dominant = false);
```

Reply:
I have made this change in my latest commit. Let me know if I missed anything.

Reply:
Could you check the [the comment style](https://github.com/NCAR/tuv-x/blob/20e73de68b72c6ad8bbb7fd5be4ab102f4761397/include/tuvx/profile.hpp#L26-L36) and my suggestions as well?